parser-typechecker/src/Unison/Server/Syntax.hs

```diff
@@ -231,8 +231,8 @@ segmentToHtml (Segment segmentText element) =
         | isFQN = nameToHtml (Name.unsafeFromText sText)
         | otherwise = L.toHtml sText
    in case ref of
-        Just r ->
-          span_ [class_ className, data_ "ref" r] content
+        Just (r, refType) ->
+          span_ [class_ className, data_ "ref" r, data_ "ref-type" refType] content
```

Comment:
Unison Share has urls with `types` or `terms` in them, we want to make sure enough information is provided to construct such urls.